httpserver/http_connection.cpp

```diff
@@ -476,8 +476,13 @@ void connection_t::process_request() {
         }
         break;
     case http::verb::get:
-        BOOST_LOG_TRIVIAL(error) << "GET requests not supported";
-        response_.result(http::status::bad_request);
+        /// Accept GET json rpc requests from the deamon
+        if (target == "/json_rpc") {
```

Comment:
Should we have a more specific endpoint for this (i.e. "/uptime_check" or something)?